app/views/ResultsPageView.scala.html

```diff
@@ -106,6 +128,15 @@ <h2 class="govuk-heading-m">@messages("resultsPage.yourDetails")</h2>
             </dd>
         </div>
     </dl>
+
+    @calculatorResult match {
```

Comment:
Wondering if using a fold method (as done in my PR) is a better option as it keeps pattern matches in one place and avoids use of `case _` to cover default cases (fold makes it explicit)

Reply:
Done